### ISSUE TITLE
Nan loss and Some important things for training

### ISSUE THREAD

**Issue by lxtGH** [playerkk/face-py-faster-rcnn#28]
When I run this repo，I got this issue
/home/lxt/study/face-py-faster-rcnn/tools/../lib/fast_rcnn/bbox_transform.py:50: RuntimeWarning: overflow encountered in exp
  pred_w = np.exp(dw) * widths[:, np.newaxis]
/home/lxt/study/face-py-faster-rcnn/tools/../lib/fast_rcnn/bbox_transform.py:51: RuntimeWarning: overflow encountered in exp
  pred_h = np.exp(dh) * heights[:, np.newaxis]
I0603 15:44:13.525365 30407 solver.cpp:218] Iteration 20 (2.86135 iter/s, 6.98971s/20 iters), loss = nan
I0603 15:44:13.525403 30407 solver.cpp:237]     Train net output #0: accuracy = 1
I0603 15:44:13.525410 30407 solver.cpp:237]     Train net output #1: loss_bbox = nan (* 1 = nan loss)
I0603 15:44:13.525416 30407 solver.cpp:237]     Train net output #2: loss_cls = 87.3365 (* 1 = 87.3365 loss)
I0603 15:44:13.525421 30407 solver.cpp:237]     Train net output #3: rpn_cls_loss = 0.683836 (* 1 = 0.683836 loss)
I0603 15:44:13.525427 30407 solver.cpp:237]     Train net output #4: rpn_loss_bbox = 0.0186136 (* 1 = 0.0186136 loss)
I0603 15:44:13.525432 30407 sgd_solver.cpp:105] Iteration 20, lr = 0.001
I0603 15:44:20.724417 30407 solver.cpp:218] Iteration 40 (2.77823 iter/s, 7.19884s/20 iters), loss = nan

**Comment by lxtGH** [comment]
Thanks for your repo ! 
Here I give my personal solution for  training this repo.
First, you should git clone the standard py-faster-rcnn repo on RBG github and Don't use the version that he give, there are some **version** problem error like the below and make. (if  meet some error , google it on Github), This is a very important thing!!!
Then, modify the python file if meet some **name** error.

**Comment by lxtGH** [comment]
Also Please change your readme.md for saving others time
